coalib/settings/SectionFilling.py

```diff
@@ -45,7 +47,12 @@ def fill_section(section, acquire_settings, log_printer, bears):
 
     # Get missing ones.
     if len(needed_settings) > 0:
-        new_vals = acquire_settings(None, needed_settings, section)
+        if len(signature(acquire_settings).parameters) is 2:
+            logging.warning('fail_acquire_settings: section parameter is '
+                            'deprecated.')
```

Comment:
you could already copy and adapt the test from `InteractionsTest` here so this message is already tested.

Reply:
ping, tests still missing in this commit